src/abstractdataframe/abstractdataframe.jl

```diff
@@ -374,28 +374,35 @@ end
 Summarize the columns of an AbstractDataFrame
 
 ```julia
-describe(df::AbstractDataFrame)
-describe(io, df::AbstractDataFrame)
+describe(df::AbstractDataFrame; colstats = [:mean, :min, :median, :max, :Nmissing, :datatype])
+describe(io, df::AbstractDataFrame; colstats = [:mean, :min, :median, :max, :Nmissing, :datatype])
 ```
 
 **Arguments**
 
 * `df` : the AbstractDataFrame
 * `io` : optional output descriptor
+* `colstats` a vector of symbols representing the summaristatistics you wish to report
```

Comment:
```
`colstats::AbstractVector{Symbol}`: the summary statistics to report

Reply:
Added. Thank you for helping me with this.